Slicesy.PizzaOrder.WebApi/Repositories/Models/Address.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Slicesy.PizzaOrder.WebApi.Repositories.Models
+{
+    [Table("Address")]
+    public class Address
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public long Id { set; get; }
+
+        [Required(ErrorMessage = "Postal code is required")]
+        public string PostalCode { set; get; }
+
+        [Required(ErrorMessage = "City is required")]
+        public string City { set; get; }
+
+        [Required(ErrorMessage = "Street is required")]
+        public string Street { set; get; }
+
+        [Required(ErrorMessage = "Building number is required")]
+        public string BuildingNumber { set; get; }
+
+        [Required(ErrorMessage = "Flat number is required")]
```

Comment:
flat number raczej nie jest obowiązkowy

Reply:
W poprzednim projekcie tak nad tym myślałem, i myślałem, i stwierdziłem, że lepiej będzie jeśli flat number w bazie będzie "-" niż nullem. Na którymś poziomie logiki się to ustawi, jeśli ktoś nie poda. Co ty na to?

Reply:
Tzn nie wiem jak to będzie na froncie. Czy nie łatwiej tam będzie sprawdzić czy w zwrotce czy flat jest nullem niż sprawdzać po stringu? Jak nie to dla mnie może być tak jak piszesz

Reply:
No właśnie często z frontu częściej dostajesz pusty string niż null, więc i tak na 90% to nie będzie null

Reply:
Aha, no to skoro tak to wygląda to oks